UC-Berkeley-Extension/global
# Header

## Asana Ticket(s)
- [Header Component](https://app.asana.com/0/0/1198192810873441/f).

## Description
Recreate existing header. Current pull request includes:
* int-headerBanner.jsp --> the structure of the banner/nav
* _common-header.scss has the vast majority of the styling. We share this one file between Global and our domestic programs sites.
* international- bhgap- and legal-studies provide for the variations from common-.

------

- Created header components including menus and search, and footer component
- Rough styleguide addition to above components
- Removed js errors
- Themed utility menu in Drupal

## Testing instructions
- `ddev start`, `drush cim`, `drush cr`
- Add utility menu items (https://global.ddev.site/admin/structure/menu/manage/utility-menu)
- Verify that the items show up on the header
- Verify that search block is on the header
- Verify that the logo shows up on the header
- Verify that the footer exists
<img width="940" alt="Screen Shot 2020-10-14 at 9 17 56 AM" src="https://user-images.githubusercontent.com/10488517/96044940-d5a03000-0e36-11eb-8cc5-882e3d01ac35.png">

## To do
- [ ] CSS styling not complete
